dashboards-observability/public/components/event_analytics/explorer/explorer.scss

```diff
@@ -21,4 +21,15 @@
 .mainContentTabs .euiResizableContainer {
   height: calc(100vh - 298px);
 }
-  
\ No newline at end of file
+
+.data_config_resizable_btn {
+  margin-left: 2px !important;
```

Comment:
This is your custom class, so why do you need important declaration?

Reply:
Important Declaration is necessary as without that the border is getting overlapped.